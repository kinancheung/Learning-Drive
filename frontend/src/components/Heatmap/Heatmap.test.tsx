import { render } from '@testing-library/react';

import { Heatmap } from '.';

describe('Heatmap', () => {
  it('should render without error', () => {
    const data = new Array(90).fill(0).map(() => Math.floor(Math.random() * 10));

    const view = render(<Heatmap data={data} />);
    expect(view.asFragment()).toMatchSnapshot();
  });
});
