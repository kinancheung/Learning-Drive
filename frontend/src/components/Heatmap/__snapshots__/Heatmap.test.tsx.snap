// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heatmap should render without error 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-0"
  >
    <div
      class="css-d9cn1v"
    >
      <div
        aria-label="01 Jan 2020: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="31 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="30 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="29 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="28 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="27 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="26 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="25 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="24 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="23 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="22 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="21 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="20 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="19 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="18 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="17 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="16 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="15 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="14 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="13 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="12 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="11 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="10 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="09 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="08 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="07 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="06 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="05 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="04 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="03 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="02 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="01 Dec 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="30 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="29 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="28 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="27 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="26 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="25 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="24 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="23 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="22 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="21 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="20 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="19 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="18 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="17 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="16 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="15 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="14 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="13 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="12 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="11 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="10 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="09 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="08 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="07 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="06 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="05 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="04 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="03 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="02 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="01 Nov 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="31 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="30 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="29 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="28 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="27 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="26 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="25 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="24 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="23 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="22 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="21 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="20 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="19 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="18 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="17 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="16 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="15 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="14 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="13 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="12 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="11 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="10 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="09 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="08 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="07 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="06 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="05 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
      <div
        aria-label="04 Oct 2019: 5 posts"
        class=" css-1kc46rb"
        data-mui-internal-clone-element="true"
      />
    </div>
  </div>
</DocumentFragment>
`;
